dspace-server-webapp/src/test/java/org/dspace/app/rest/GroupRestRepositoryIT.java

```diff
@@ -450,6 +459,109 @@ private void runPatchMetadataTests(EPerson asUser, int expectedStatus) throws Ex
         new MetadataPatchSuite().runWith(getClient(token), "/api/eperson/groups/" + group.getID(), expectedStatus);
     }
 
+    @Test
+    public void patchGroupName() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group group = GroupBuilder.createGroup(context).withName("Group").build();
+        context.restoreAuthSystemState();
+        String token = getAuthToken(admin.getEmail(), password);
+
+        String requestBody
+                = "      [\n"
+                + "        {\n"
+                + "          \"op\": \"replace\",\n"
+                + "          \"path\": \"/name\",\n"
+                + "          \"value\": \"new name\"\n"
+                + "        }\n"
+                + "      ]";
```

Comment:
You should not need to manually hardcode the JSON request.  We have a `getPatchContent()` method that will generate this for you.  Here's another replace example from another IT: https://github.com/DSpace/DSpace/blob/master/dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java#L411-L414